source/aggregation-pipeline-builder.txt

```diff
@@ -183,6 +183,39 @@ Open a Saved Pipeline
    To learn more, see 
    :ref:`viewing saved aggregations<saved-query-view>` for details.
 
+.. _set-maxtime-ms-agg:
+
+Set ``maxTimeMS`` for Aggregations Queries
+------------------------------------------
+
+Use the ``maxTimeMS`` parameter on the aggregation tab to put an 
+upper time limit in milliseconds on the aggregation pipelines that 
+run in Compass.
+
+To enable ``maxTimeMS`` on the aggregation tab:
+
+.. procedure::
+   :style: normal
+
+   .. step:: In the aggregation pipeline pane, click the 
+      :guilabel:`More Options`.
+
+      .. figure:: /images/compass/agg-builder-click-more-options.png
+         :alt: Toggle more options
+
+   .. step:: Enter a value next to :guilabel:`Max Time MS` to set a 
+      maximum amount of time in milliseconds that an aggregation 
+      pipeline can run. For a ``5`` second limit enter ``5000``.
+
+      .. figure:: /images/compass/agg-builder-enter-maxtime-ms.png
+         :alt: Enter a maximum timeout value 
+
+   .. step:: Any aggregation pipeline that goes over the time limit 
```

Comment:
[question/suggestion] I feel like this statement about the error message would be better suited outside the procedure directive / not as a step since it's not particularly a step you have to take to set the `maxTimeMS` option. 

Can take it out of the step and instead say something along the lines of: 
"If your aggregation operation goes over the time limit, |compass-short| raises the following error: "

Reply:
I think you are correct I have staged it both ways, its not part of the procedure and more of a demonstration piece of information. I have removed this step and updated.